tests/src/test/java/alluxio/hadoop/HdfsFileInputStreamIntegrationTest.java

```diff
@@ -53,10 +59,11 @@
 
   @BeforeClass
   public static final void beforeClass() throws Exception {
-    FileSystem fileSystem = sLocalAlluxioClusterResource.get().getClient();
+    sFileSystem = sLocalAlluxioClusterResource.get().getClient();
     FileSystemTestUtils
-        .createByteFile(fileSystem, "/testFile1", WriteType.CACHE_THROUGH, FILE_LEN);
-    FileSystemTestUtils.createByteFile(fileSystem, "/testFile2", WriteType.THROUGH, FILE_LEN);
+        .createByteFile(sFileSystem, "/testFile1", WriteType.CACHE_THROUGH, FILE_LEN);
```

Comment:
These test files are shared mutable resources, so they shouldn't be created statically because then tests will interfere with each other. Could we move the test file creation to the `@Before`? Also, we should name them better as `inMemoryFile` and `ufsOnlyFile`, and define those names as static constants. To avoid collisions we could either delete the files in `@After` or add salt to make the names unique during each test, e.g. adding a random number from a large range or a UUID.

If we make these changes you won't need to add testFile3 any longer